hw/dv/tools/dvsim/riviera.hjson

```diff
@@ -23,6 +23,7 @@
                "-lib {sv_flist_gen_dir}/work",
                "+UVM_TESTNAME={uvm_test}",
                "+UVM_TEST_SEQ={uvm_test_seq}",
+               "-sv_lib \"{ALDEC_PATH}/bin/uvm_1_2_dpi\"",			   
```

Comment:
Does Riviera support passing UVM DPI-C sources for native compilation?

Reply:
dpi need to be build with ccomp and its result "so/dll" library should be passedas parameter to asim -sv_lib command. 
ccomp is just a wrapper for actual compiler (gcc or visual). It deal with extra compiler arguments so user do not need to bother.